gpMgmt/bin/pythonSrc/PyGreSQL-4.0/pgmodule.c

```diff
@@ -55,7 +122,7 @@ typedef int Py_ssize_t;
 #endif
 
 /* taken from fileobject.c */
-#define BUF(v) PyString_AS_STRING((PyStringObject *)(v))
+#define BUF(v) PyBytes_AS_STRING((PyStringObject *)(v))
```

Comment:
```suggestion
#define BUF(v) PyBytes_AS_STRING((PyObject *)(v))
```
because PyBytes_AS_STRING is declared as
```
static inline char* PyBytes_AS_STRING(PyObject *op)
```